client-frameworks-support/client-support-angular/package.json

```diff
@@ -49,5 +49,8 @@
   },
   "engines": {
     "node": ">=18"
+  },
+  "publishConfig": {
+    "directory": "dist/client-support-angular"
```

Comment:
I assumed this was mostly about the E2E test app, would this change also affect the release process of support-angular package. I am not sure. why was this needed here

Reply:
Yes, that's right. It's necessary otherwise the wrong package (this [one](https://github.com/SAP/luigi/tree/main/client-frameworks-support/client-support-angular) which is one level up) will be installed for the e2e-test-application.

Codewise it should not have an effect on the release, see [here](https://github.com/SAP/luigi/blob/main/scripts/npmPublish.sh#L147).
In addition I also did a nightly release which was successful.